tower.yml

```diff
@@ -1,5 +1,59 @@
 reports:
   multiqc_report.html:
     display: "MultiQC HTML report"
-  samplesheet.csv:
-    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
+  "**/fastqc/raw/*.html":
```

Comment:
@LilyAnderssonLee are we missing any new log files for the three new tools?

Reply:
Yes, updated #534